src/i18n/en-US/index.js

```diff
@@ -37,9 +37,11 @@ export default {
       empty: 'No files available in project.',
       newFile: 'New file',
       newFolder: 'New folder',
+      addFile: 'Add',
       file: {
         create: 'File is created &#129395;!',
         delete: 'File is deleted.',
+        add: 'File is added &#129395;!',
```

Comment:
no wink for the delete. so sad.

Reply:
![image](https://user-images.githubusercontent.com/7418386/199736892-adb7a4b3-8be0-4449-ae1d-708b66b94a90.png)

Reply:
I did not add it on purpose, @Zorin95670 what do you think?